src/framework/uicomponents/qml/MuseScore/UiComponents/FlatButton.qml

```diff
@@ -49,7 +51,11 @@ FocusableControl {
         }
     }
 
-    keynav.onTriggered: root.clicked()
+    keynav.onTriggered: {
+        if (root.isClickOnKeyNavTriggered) {
```

Comment:
Could you explain why we should track this, what would be if we'll just emit clicked() signal?

Reply:
From a chat with Martin:
```
Me:
There was some inconsistency with the note input button.
Now implemented:
* Click - toggle the note input mode
* PressAndHold - show a menu with modes

The document on keyboard navigation says:
* Press Enter - show a menu with modes

It turns out that the press on Enter corresponds to the PressAndHold mouse action, although in all other places, press on Enter corresponds to the Click mouse action.
Will we keep this exception?
Or reconsider what should be when we click on the note input button?

Martin:
Let me think for a moment...
Peter Jonas and I chatted about this a lot. We felt it was better for it to open the context menu instead. This would force a two-step action. (Becuase blind people would learn fairly quickly that a double-tap would be necessary. But it is much more likely that they'll simply use 'N' to toggle it on/off)

So if focused on the Note Input button (turned off or on) and pressing Enter, the context menu appears and the focus moves to the selected item in the list and the user has to press enter again to activate it, which closes the context menu.

Does that make sense?
So I would say it is good to keep the exception. Peter Jonas - am I talking crazy, or is that right?

Peter:
 that's sounds right to me. @igorkorsukov, see this question from Casper and my response for details: https://discord.com/channels/818804595450445834/818848561047273544/829707627386503190
```